### PR TITLE
Sec Shades Returns to Staff

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

Sec shades are given back to Security Officers, Warden, and HoS.
## Why / Balance

Locking sec shades behind tier 2 research made it so Sec was doomed to never have them again.
## Media
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**
:cl:
- add: Added sec sunglasses to security officers, warden, and head of security.

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Lockers/security.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@
       - id: RubberStampWarden
       - id: DoorRemoteArmory
       - id: HoloprojectorSecurity
-      - id: ClothingEyesHudSecurity
+      - id: ClothingEyesHudSecurity #Delta-V: Sechud Scarcity
```
Comment (DebugOk) [inline Resources/Prototypes/Catalog/Fills/Lockers/security.yml:52]: So what exactly is getting changed besides the comment?

File: Resources/Prototypes/Catalog/Fills/Lockers/security.yml [hunk 2]
```diff
@@ -92,7 +92,7 @@
       - id: ClothingShoesBootsJack
       - id: WeaponMeleeNeedle
         prob: 0.1
-      - id: ClothingEyesHudSecurity
+      - id: ClothingEyesHudSecurity #Delta-V: Sechud Scarcity
```
Comment (DebugOk) [inline Resources/Prototypes/Catalog/Fills/Lockers/security.yml:95]: Same deal here

File: Resources/Prototypes/Roles/Jobs/Security/head_of_security.yml [hunk 3]
```diff
@@ -50,7 +50,8 @@
     back: ClothingBackpackHOSFilled
     shoes: ClothingShoesBootsCombatFilled
     outerClothing: ClothingOuterCoatHoSTrench
-    eyes: ClothingEyesGlassesSunglasses
+    eyes: ClothingEyesGlassesSecurity # Delta-V: Replaced Regular Sunglasses
+    # eyes: ClothingEyesGlassesSunglasses # Delta-V: Replaced
```
Comment (DebugOk) [inline Resources/Prototypes/Roles/Jobs/Security/head_of_security.yml:54]: Just replace the entire line and say you replaced it, this doesnt help

**Review by DebugOk — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Lockers/security.yml (hunk 1 above)
Comment (ps3moira) [reply Resources/Prototypes/Catalog/Fills/Lockers/security.yml:52]: Originally, I had it removed. It ended up causing some issues and I'm currently working on trying to fix it.

**Comment by DebugOk** [comment]
This pr may be closed if it stays derelict

**Comment by VMSolidus** [comment]
I'm going to close this PR as unecessary. In practice Security gets the Sec Shades in nearly every round, since they're cheap to make, good wardens get on top of making them, and Epi tends to prioritize early security research so that they can take advantage of Oracle's ~50% chance of asking for Security techfab stuff.

**Pull request closed without merging**